src/components/Translation.tsx

```diff
@@ -1,12 +1,26 @@
 import React from "react"
-import { Trans } from "gatsby-plugin-react-i18next"
+import { useTranslation } from "gatsby-plugin-react-i18next"
+import htmr from "htmr"
 
-interface Props extends React.ComponentProps<typeof Trans> {
+import Link from "./Link"
+
+interface Props {
   id: string
 }
 
+const transform = {
+  a: Link,
+}
+
 // Wrapper on <FormattedHTMLMessage /> to always fallback to English
```

Comment:
Seems this explanation is now out of date (given we swapped i18n libs)? Does it still fallback to English?

Reply:
Correct. Yea, I'll update that comment. And yes, it fallback to English.